### DISCUSSION TITLE
General maintenance question

### DISCUSSION THREAD

**Discussion by jonahallibone** [react-grid-layout/react-grid-layout#2065] [General]
Hi -- just wondering if this project is still maintained? I know there's been a PR or two merged since the last release and bunch of the docs recently broke, however, it seems there's been no movement on fixing the docs are addressing issues. Understandably this is free software maintained at the time and expense of the maintainers, but I have just created a prototype for my company using this software and just want to know what the future of this repo looks like.

I am really enjoying my time with this as well, and would love to eventually be a contributor as well. Let me know if there's anything I can do to help!

Thanks!

**Comment by airtonix** [comment]
2 weeks no answer. project is dead. I'd move on.
